LICENSE

```diff
@@ -1,6 +1,7 @@
 MIT License
 
 Copyright (c) 2021 Hellowlol and contributers
+Copyright (c) 2023 Svein Seldal, @sveinse
```

Comment:
Lets just have one copywrite line:
.... 2023 @Hellowlol , @sveinse and contributers

Reply:
Agreed. I had them separate because I alone can't make the copyright 2023 claim on your behalf.